README.md

```diff
@@ -1 +1,112 @@
 # ruby_oop-school-library_m4_w3
+
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
+
+"Ruby OOP school library" is  a tool to record what books are in the library and who borrows them. The app  allow you to:
```

Comment:
- [ ] Let's add  .vscode to the gitignore file so we have a clean repository.

Reply:
:+1: